client/my-sites/plans/jetpack-plans/types.ts

```diff
@@ -57,6 +57,7 @@ export type PlanGridProducts = {
 };
 
 export interface JetpackFreeProps {
+	fullWidth?: boolean;
```

Comment:
This property is what triggers the activation of the `--full-width` style variant I mentioned earlier.